tests/unit/Directors.spec.ts

```diff
@@ -365,291 +552,243 @@ xdescribe('Directors as a COOP', () => {
+    expect(vm.allDirectors[2].officer.middleInitial).toEqual('S')
+    expect(vm.allDirectors[2].officer.lastName).toEqual('Swanson')
+  })
 
-    // shows list of all directors in the UI, in reverse order in which they are in the json
-    expect(directorListUI.length).toEqual(2)
-    expect(directorListUI[0].textContent).toContain('Griffin')
-    expect(directorListUI[0].textContent).toContain('mailing_address city')
-    expect(directorListUI[0].textContent).toContain('4321')
-    expect(directorListUI[1].textContent).toContain('Joe')
-    expect(directorListUI[1].textContent).toContain('Glasgow')
-    expect(directorListUI[1].textContent).toContain('1234')
-    expect(directorListUI[1].querySelector('.same-address').textContent).toContain('Same as Delivery')
+  it('loads the director addresses properly', () => {
+    // spot check one complete address
+    const deliveryAddress = vm.allDirectors[2].deliveryAddress
+    expect(deliveryAddress.streetAddress).toBe('street-del-1')
+    expect(deliveryAddress.streetAddressAdditional).toBe('additional-del-1')
+    expect(deliveryAddress.addressCity).toBe('city-del-1')
+    expect(deliveryAddress.addressCountry).toBe('UK')
+    expect(deliveryAddress.postalCode).toBe('UKU KUK')
+    expect(deliveryAddress.addressRegion).toBe('Scotland')
+    expect(deliveryAddress.deliveryInstructions).toBe('go to the loch')
+
+    const mailingAddress = vm.allDirectors[2].mailingAddress
+    expect(mailingAddress.streetAddress).toBe('street-mail-1')
+    expect(mailingAddress.streetAddressAdditional).toBe('additional-mail-1')
+    expect(mailingAddress.addressCity).toBe('city-mail-1')
+    expect(mailingAddress.addressCountry).toBe('UK')
+    expect(mailingAddress.postalCode).toBe('UKU KUK')
+    expect(mailingAddress.addressRegion).toBe('Scotland')
+    expect(mailingAddress.deliveryInstructions).toBe('go to the highlands')
+  })
 
-    // shows "cease" button, indicating this is an active director, ie: starting state for list
-    expect(directorListUI[1].innerHTML).toContain('<span>Cease</span>')
-    expect(directorListUI[0].innerHTML).toContain('<span>Cease</span>')
+  it('displays the list of directors', () => {
+    const directorListUI = wrapper.findAll('.director-list-item .director-list__first-name')
+    expect(directorListUI.length).toBe(3)
+
+    // spot check the director names
+    expect(directorListUI.at(0).text()).toBe('Gary')
+    expect(directorListUI.at(1).text()).toBe('Peter')
+    expect(directorListUI.at(2).text()).toBe('Sam')
+
+    // verify directors are active (initial state)
+    // ("Cease" button indicates this is an active director)
+    expect(vm.isActive(vm.allDirectors[0])).toBe(true)
+    expect(vm.isActive(vm.allDirectors[1])).toBe(true)
+    expect(vm.isActive(vm.allDirectors[2])).toBe(true)
   })
 
   it('disables buttons/actions when instructed by parent component', async () => {
     // clear enabled prop
-    vm.componentEnabled = false
```

Comment:
This is another one of those things that I don't know how it worked before. Theoretically you're not able to assign a prop, but ...?
Anyway, I fixed it.